src/components/AddressContainer/index.tsx

```diff
@@ -101,12 +103,17 @@ const AddressContainer = () => {
 
       {/* Not register to react-hook-form because we only want to
           use this value for the no-js scenario. */}
-      <input type="hidden" aria-hidden={true} name="page" value="location" />
-      <input
+      <FormField
+        id="hidden-location-page"
+        type="hidden"
+        name="page"
+        defaultValue="location"
+      />
+      <FormField
+        id="hidden-form-values"
         type="hidden"
-        aria-hidden={true}
```

Comment:
Why remove this?

Reply:
Internally, `FormField` renders a DS `Input` component of type "hidden". That component renders `aria-hidden={true}` now so no need to explicitly add it here.